modules/installation-and-upgrade/pages/container-deployment/suma/server-deployment-vm-suma.adoc

```diff
@@ -168,7 +174,7 @@ mgr-storage-server /dev/nvme1n1 /dev/nvme2n1
 +
 [NOTE]
 ====
-This command will create the persistent storage volumes at [path]``/var/lib/containers/storage/volumes``.
+This command will move the persistent storage volumes at [path]``/var/lib/containers/storage/volumes`` to specified storage devices.
```

Comment:
From what I understand this should be creates rather than move. This tool is creating these volumes. Its not really moving anything. Perhaps someone can verify?

Reply:
Probably both.  Because we now do it after deployment, it creates the storage device, then moves any already existing data (volume) to the storage device,  and mounts it (as the volume).  I'm neither sure about the sequence of these commands nor about the terminology we have to use here.

Ondrej proposed such a change here: https://github.com/uyuni-project/uyuni-docs/pull/3385#discussion_r1812321720

Reply:
@jcayouette can we keep the phrase as-is now or shall we address both ways more details.  In any case, I'd propose to merge this now, and create a separate PR as needed.  The new PR then could all places where we now have such a note and turn it into a snippet (or something similar).

Reply:
Im fine with your change. Thank you karl.